src/crop_image.py

```diff
@@ -1,69 +1,122 @@
-#!/usr/bin/env python
 
+'''
+Created on March 15th, 2018
+author: Michael Rodriguez
+sources: https://www.blog.pythonlibrary.org/2017/10/03/how-to-crop-a-photo-with-python/
+description: Crops images of a given object class to feed into neural network.
+'''
+# !/usr/bin/env python
 from os import listdir
-from os.path import isfile, join, splitext
+from os.path import splitext
 from PIL import Image
-import numpy as np
-import cv2
-import glob
+import os
+import sys
+import argparse
 
 
 def crop(image_path, coords, saved_location):
     """
-    @param image_path: The path to the image to edit
-    @param coords: A tuple of x/y coordinates (x1, y1, x2, y2)
-    @param saved_location: Path to save the cropped image
+    crop - Image cropping function
```

Comment:
Function headers https://github.com/julianweisbord/Deep-Learning-For-Object-Recognition-on-a-Mobile-Robot/blob/master/src/learning/model.py